src/main/java/jalau/at18/azul/Pyramide.java

```diff
@@ -0,0 +1,19 @@
+package jalau.at18.azul;
+
+public class Pyramide {
+    private final int pyramideSize = 5;
+    private final int pyramideLimit = 4;
+    private char[][] tile = new char[pyramideSize][pyramideSize];
+
+    public void initializePyramide() {
+        for (int row = 0; row < pyramideSize; row++) {
+            for (int column = 0; column < pyramideSize; column++) {
+                tile[row][column] = row + column >= pyramideLimit ? 'O' : ' ';
```

Comment:
What do `'0'` and `' '` mean?

Reply:
at the moment it is a logic to fill the matrix in shape of a pyramid until using the enum of the tiles

Reply:
ok, thanks for clarification